src/app.js

```diff
@@ -6,8 +6,256 @@ import _ from 'underscore';
 import './css/foundation.css';
 import './css/style.css';
 
+// CLASSES
+import Trip from 'app/models/trip';
+import TripList from 'app/collections/trip_list';
+import Reservation from 'app/models/reservation';
+
 console.log('it loaded!');
 
+// variables
+const TRIP_FIELDS = ['name', 'continent', 'about', 'category', 'weeks', 'cost'];
+const RESERVATION_FIELDS = ['name', 'age', 'email'];
+const CONTINENTS = ['Africa', 'Antartica', 'Asia', 'Australasia', 'Europe', 'North America', 'South America'];
+// create variable for using in document.ready
+const tripList = new TripList();
+
+//define templates
+let tripsTemplate;
+let backTemplate;
+let headTemplate;
+let tripTemplate;
+let reserveTemplate;
+let addTripTemplate;
+let statusMessage;
+
+//define global variable
+var showFirst = false;
+
+//Validation handling
+const handleValidationFailures = function(errors) {
+  for (let field in errors) {
+    for (let problem of errors[field]) {
+      reportStatus('errors', `${ field }: ${ problem }`);
+    }
+  }
+};
+
+//clear status messages
+const clearStatus = function() {
+  $('#status-messages ul').html('');
+  $('#status-messages').hide();
+}
+
+//add a new status message
+const reportStatus = function(status, message) {
+  const statusHTML = `<li class="${ status }">${ message }</li>`;
+  $('#status-messages ul').append(statusHTML);
+  $('#status-messages').show();
+}
+
+//get all trips
+const getTrips = function(tripList) {
+  let tableHead = headTemplate();
+
+  const tripTableElement = $('#trip-list');
+  tripTableElement.html('');
+  for (let trip of tripList.models) {
+    const generatedHTML = $(tripsTemplate(trip.attributes));
+    // generatedHTML.on('click', (event) => {
+    //   getTrip(trip);
+    // });
+    tripTableElement.append(generatedHTML);
+  }
+
+  //hide header and #trips button and show nav section
+  $('h1').hide();
+  $('#trips').hide();
+  $('nav').show();
+
+  //fill out .trips section
+  $('.trips thead').html(tableHead);
+  $('.trip').show();
+
+  //fill out .trip section with first trip of collection
+  if (showFirst === false) {
+    let trip = tripList.first();
+    trip.on('change', getTrip);
+    trip.fetch();
+    showFirst = true;
+  }
+
+  //sort by columns in trips table
+  TRIP_FIELDS.forEach((field) => {
+    const headerElement = $(`th.sort.${ field }`);
+    headerElement.on('click', (event) => {
+      console.log(`sorting trips by ${ field }`);
+      tripList.comparator = field;
+      tripList.sort();
+    });
+  });
+
+  //provide visual feedback for sorting
+  $('th.sort').removeClass('sort-field');
+  $(`th.sort.${ tripList.comparator }`).addClass('sort-field');
+
+
+}; // end of getTrips
+
+const getTrip = function(trip) {
+  console.log('get trip');
+  console.log(trip);
+  let generatedHTML = tripTemplate(trip.attributes);
+  $('#trip').html(generatedHTML);
+  $('#reservation').append(statusMessage());
+  $('#reservation').append(reserveTemplate(trip));
```

Comment:
Clicking on a trip description will keep appending reservation forms to the `#reservation` block.  Instead consider setting the `html` inside it as opposed to appending.